pkg/dispatchcli/cmd/uninstall.go

```diff
@@ -35,10 +34,10 @@ var (
 	uninstallNamespace       = i18n.T(``)
 	uninstallRemoveCertFiles = false
 
-	defaultUninstallConfig = installConfig{
-		Namespace: "dispatch",
-		Hostname:  "dispatch.vmware.com",
-	}
+	// defaultUninstallConfig = installConfig{
```

Comment:
So is the config a required arg now?

Reply:
👍  I am working on to have a default config.yaml